onnxruntime/python/tools/symbolic_shape_infer.py

```diff
@@ -2406,5 +2429,16 @@ def parse_arguments():
         args.verbose,
     )
     if args.output and out_mp:
-        onnx.save(out_mp, args.output)
+        if args.save_as_exteranl_data:
```

Comment:
still have a typo here

Reply:
Thanks for checking.
Fixed this typo and run the script locally to double check.